autogalaxy/interferometer/model/analysis.py

```diff
@@ -352,6 +322,9 @@ def save_attributes(self, paths: af.DirectoryPaths):
             hdu=self.dataset.real_space_mask.hdu_for_output,
             prefix="dataset",
         )
+        paths.save_json(
+            "transformer_class", to_dict(self.dataset.transformer.__class__), "dataset"
+        )

```

Comment:
Maybe we should have paths implicitly apply 'to_dict' to any object it is passed which is not a dictionary or list